dare.go

```diff
@@ -44,9 +44,9 @@ const (
 
 var (
 	errMissingHeader      = errors.New("dare: incomplete header")
```

Comment:
Then lets fix it properly @aead and then we can rename them appropriately. It is not correct to add errors with a different name and then have a `package-name` different.